lib/core/include/irods/objInfo.h

```diff
@@ -151,9 +151,6 @@ typedef struct DataObjInfo {
     int  dataAccessInx;
     int  writeFlag;
     char destRescName[NAME_LEN];
-    // The backupResc feature has been deprecated and will be removed in a future release. It is not being marked as
-    // deprecated for compilation purposes because we cannot change this struct and it is used everywhere.
-    char backupRescName[NAME_LEN];
```

Comment:
Can't remove this because DataObjInfo's are sent over the wire. Removing this may break federation.

Reply:
Reverted.